src/hotspot/os/linux/cgroupSubsystem_linux.cpp

```diff
@@ -494,7 +494,7 @@ int CgroupSubsystem::active_processor_count() {
   int period = cpu_period();
 
   if (quota > -1 && period > 0) {
-    quota_count = ceilf((float)quota / (float)period);
+    quota_count = (int)ceilf((float)quota / (float)period);
```

Comment:
Should be OK except for the pathological case of quota near INT_MAX and period == 1.

Reply:
Yes, exactly, and we have hit similar bugs before.  But the bug seems harmless here.

Reply:
So, comment?  What should it say?

Reply:
I think it makes sense to have something like checked_cast here, maybe call it double_to_int(), that checks against INT_MIN and INT_MAX.  Also, using double instead of float removes the round-up problem as long as the input is <= 32 bits.

Reply:
I'm going to revert this change to fix at a later time.